astropy/io/registry.py

```diff
@@ -318,12 +320,20 @@ def register_identifier(data_format, data_class, identifier, force=False):
         _identifiers[(data_format, data_class)] = identifier
     else:
         raise IORegistryError("Identifier for format '{0}' and class '{1}' is "
-                              'already defined'.format(data_format,
-                                                       data_class.__name__))
+                              'already defined.'.format(data_format,
+                                                        data_class.__name__))
 
 
 def identify_format(origin, data_class_required, path, fileobj, args, kwargs):
-    # Loop through identifiers to see which formats match
+    """Loop through identifiers to see which formats match.
+
+    For the parameter description see `register_identifier`.
```

Comment:
Sorry I didn't notice this earlier, but the parameters for `register_identifier` are mostly different from here.  To complete the mission of this PR you need to dig in and document these parameters.